core/nox.py

```diff
@@ -35,6 +35,7 @@ def unit_tests(session, python_version):
         'pytest',
         'pytest-cov',
         'grpcio >= 1.0.2',
+        'google-gax >= 0.15, < 0.16.dev',
```

Comment:
ugh I do not feel good about this, but it's fine for now. I'll remove it once I move gax's error functionality into here.